src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/ProbationTeamsService.kt

```diff
@@ -19,10 +19,10 @@ class ProbationTeamsService(
 ) {
   @Cacheable(CacheConfiguration.PROBATION_TEAMS_CACHE)
   fun getProbationTeams(enabledOnly: Boolean) = if (enabledOnly) {
-    probationTeamRepository.findAllByEnabledIsTrue().toModel()
+    probationTeamRepository.findAllByEnabledIsTrue()
   } else {
-    probationTeamRepository.findAll().filter(ProbationTeam::isReadable).toModel()
-  }
+    probationTeamRepository.findAll().filter(ProbationTeam::isReadable)
+  }.sortedWith(compareBy({ !it.enabled }, { it.description })).toModel()
 
```

Comment:
What order will the teams show up? Not listed at the top or the bottom?

Reply:
At the bottom.